adapta/storage/database/trino_sql.py

```diff
@@ -106,7 +106,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self._connection.close()
         self._engine.dispose()
 
-    def query(self, query: str, batch_size: int = 1000) -> Iterator[DataFrame]:
```

Comment:
i see all iterators were changed to iterables - why?

Reply:
reverted this